src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,61 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
```

Comment:
However, I'm wondering why this is included 8a1980f0542712c38a944412b33577dffac215ca, I would prefer to have a dedicated commit for these tests (maybe even PR as it seems it is completely unrelated)